### PR TITLE
refactor: cosmos-sdk modules

### PR DESCRIPTION
**Pull request by nowooj**

### CODE REVIEW

File: content/en/docs/develop/core-modules/bank.md [hunk 1]
```diff
@@ -6,23 +6,94 @@ weight: 40
...
+XPLA Chain's bank module inherits from the Cosmos SDK's [`bank`](https://docs.cosmos.network/v0.45/modules/bank/) module. This document is a stub and covers mainly important XPLA Chain-specific notes about how it is used.
 {{< /alert >}}
 
 The bank module is the base transactional layer of the XPLA Chain. This module allows assets to be sent from one `Account` to another. The bank module defines the following types of send transactions: `MsgSend` and `MsgMultiSend`.
 
+The bank module is responsible for handling multi-asset coin transfers between accounts and tracking special-case pseudo-transfers which must work differently with particular kinds of accounts (notably delegating/undelegating for vesting accounts). It exposes several interfaces with varying capabilities for secure interaction with other modules which must alter user balances.
+
+In addition, the bank module tracks and provides query support for the total supply of all assets used in the application.
+
+This module will be used in the Cosmos Hub.
```
Comment (jhlee-young) [inline content/en/docs/develop/core-modules/bank.md]: plz update the network name

Comment (nowooj) [reply content/en/docs/develop/core-modules/bank.md]: https://github.com/xpladev/docs/pull/13/commits/5157a0daf4b64072bce960838115cf552c632aed

File: content/en/docs/develop/core-modules/staking.md [hunk 2]
```diff
@@ -6,176 +6,604 @@ weight: 140
...
+concern for the changing public key.
+
+`ValidatorByConsAddr` is an additional index that enables lookups for slashing.
+When Tendermint reports evidence, it provides the validator address, so this
+map is needed to find the operator. Note that the `ConsAddr` corresponds to the
+address which can be derived from the validator's `ConsPubKey`.
+
+`ValidatorsByPower` is an additional index that provides a sorted list of
+potential validators to quickly determine the current active set. Here
+ConsensusPower is validator.Tokens/10^6 by default. Note that all validators
```
Comment (jhlee-young) [inline content/en/docs/develop/core-modules/staking.md]: 10^6 -> 10^18

File: content/en/docs/develop/core-modules/upgrade.md [hunk 3]
```diff
@@ -6,24 +6,124 @@ weight: 150
 {{< alert >}}
 **Note**
 
-XPLA Chain's upgrade module inherits from the Cosmos SDK's [`upgrade`](https://docs.cosmos.network/master/modules/upgrade/) module. This document is a stub and covers mainly important XPLA Chain-specific notes about how it is used.
+XPLA Chain's upgrade module inherits from the Cosmos SDK's [`upgrade`](https://docs.cosmos.network/v0.45/modules/upgrade/) module. This document is a stub and covers mainly important XPLA Chain-specific notes about how it is used.
 {{< /alert >}}
 
-The upgrade module allows live chains to be upgraded with breaking changes smoothly.
+`x/upgrade` is an implementation of a Cosmos SDK module that facilitates smoothly upgrading a live Cosmos chain to a new (breaking) software version. It accomplishes this by providing a `BeginBlocker` hook that prevents the blockchain state machine from proceeding once a pre-defined upgrade block height has been reached.
```
Comment (jhlee-young) [inline content/en/docs/develop/core-modules/upgrade.md]: remaining network name 👆

Comment (jhlee-young) [reply content/en/docs/develop/core-modules/upgrade.md]: and the network name should be capital letters, XPLA Chain

File: content/en/docs/develop/core-modules/governance.md [hunk 4]
```diff
@@ -17,122 +17,292 @@ To learn how to vote with your staked XPLA or submit proposals, visit the [XPLA
...
-	DepositEndTime time.Time `json:"deposit_end_time" yaml:"deposit_end_time"` // Time that the Proposal would expire if deposit amount isn't met
-	TotalDeposit   sdk.Coins `json:"total_deposit" yaml:"total_deposit"`       // Current deposit on this proposal. Initial value is set at InitialDeposit
+_Note: from the UI, for urgent proposals we should maybe add a ‘Not Urgent’
+option that casts a `NoWithVeto` vote._
 
-	VotingStartTime time.Time `json:"voting_start_time" yaml:"voting_start_time"` // Time of the block where MinDeposit was reached. -1 if MinDeposit is not reached
-	VotingEndTime   time.Time `json:"voting_end_time" yaml:"voting_end_time"`     // Time that the VotingPeriod for this proposal will end and votes will be tallied
+#### Weighted Votes
+
+[ADR-037](../../../docs/architecture/adr-037-gov-split-vote.md) introduces the weighted vote feature which allows a staker to split their votes into several voting options. For example, it could use 70% of its voting power to vote Yes and 30% of its voting power to vote No.
```
Comment (jhlee-young) [inline content/en/docs/develop/core-modules/governance.md]: broken link

File: content/en/docs/develop/core-modules/governance.md [hunk 5]
```diff
@@ -246,19 +420,31 @@ for finishedProposalID in GetAllFinishedProposalIDs(block.Time)
 The subspace for the Governance module is `gov`.
 
 ```go
+// DepositParams defines the params for deposits on governance proposals.
 type DepositParams struct {
-	MinDeposit       sdk.Coins     `json:"min_deposit,omitempty" yaml:"min_deposit,omitempty"`
-	MaxDepositPeriod time.Duration `json:"max_deposit_period,omitempty" yaml:"max_deposit_period,omitempty"` //  Maximum period for Atom holders to deposit on a proposal. Initial value: 2 months
+	//  Minimum deposit for a proposal to enter voting period.
+	MinDeposit github_com_cosmos_cosmos_sdk_types.Coins `protobuf:"bytes,1,rep,name=min_deposit,json=minDeposit,proto3,castrepeated=github.com/cosmos/cosmos-sdk/types.Coins" json:"min_deposit,omitempty" yaml:"min_deposit"`
+	//  Maximum period for Atom holders to deposit on a proposal. Initial value: 2
```
Comment (jhlee-young) [inline content/en/docs/develop/core-modules/governance.md]: denom update is needed

File: content/en/docs/develop/core-modules/upgrade.md (hunk 3 above)
Comment (nowooj) [reply content/en/docs/develop/core-modules/upgrade.md]: https://github.com/xpladev/docs/pull/13/commits/5157a0daf4b64072bce960838115cf552c632aed

File: content/en/docs/develop/core-modules/staking.md (hunk 2 above)
Comment (nowooj) [reply content/en/docs/develop/core-modules/staking.md]: https://github.com/xpladev/docs/pull/13/commits/55e658d446ba9cab87cae243170fc53b34edaffc

File: content/en/docs/develop/core-modules/governance.md (hunk 5 above)
Comment (nowooj) [reply content/en/docs/develop/core-modules/governance.md]: https://github.com/xpladev/docs/pull/13/commits/6155517bc6257cdd88fa7db09b9645b643c5956d

File: content/en/docs/develop/core-modules/governance.md (hunk 4 above)
Comment (nowooj) [reply content/en/docs/develop/core-modules/governance.md]: https://github.com/xpladev/docs/pull/13/commits/cb8965ea23a4e4978be6fcb0c1bccd478671b7d0

**Review by jhlee-young — APPROVED** [review]
LGTM

File: content/en/docs/develop/core-modules/bank.md [hunk 6]
```diff
@@ -6,23 +6,94 @@ weight: 40
...
+type ModuleAccount interface {
+  auth.Account               // same methods as the Account interface
+
+  GetName() string           // name of the module; used to obtain the address
+  GetPermissions() []string  // permissions of module account
+  HasPermission(string) bool
+}
+```
+
+> **WARNING!**
```
Comment (jhlee-young) [inline content/en/docs/develop/core-modules/bank.md]: Nit: suggest using this format `{{< alert context="warning" >}}`

Comment (nowooj) [reply content/en/docs/develop/core-modules/bank.md]: https://github.com/xpladev/docs/pull/13/commits/f9fe3f2d179d843f6fc7af01d600ecbdb7f50680

**Review by caramis — APPROVED** [review]
LGTM

**Pull request merged**